CoffeeChat/SceneDelegate.swift

```diff
@@ -30,6 +30,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             window.rootViewController = navigationController
             return
         }
+        let navigationController = UINavigationController(rootViewController: LoginViewController())
+        window.rootViewController = navigationController
```

Comment:
why do we need this here? Seems like it's already handled by the else part of `if signIn.hasPreviousSignIn()`

Reply:
oh lmao don't need. I forgot to take out but I had this so that we always go to login page on launch for when I was programming heh